.github/workflows/ci.yaml

```diff
@@ -217,7 +217,7 @@ jobs:
         run: mix deps.clean --all && mix deps.get
 
       - name: Mix setup
-        run: mix setup
+        run: mix do ecto.drop, event_store.drop, ecto.create, ecto.migrate, setup
```

Comment:
Did we need this `drop` thing?
The database should be coming empty.
Just to avoid not needed steps, as we are aiming to reduce the execution time at some point